src/generated/resources/openapi.json

```diff
@@ -33,6 +33,15 @@
         }
       }
     },
+    "/api/metadata/security/v2alpha1/authorize" : {
+      "put" : {
+        "responses" : {
+          "204" : {
+            "description" : "No Content"
+          }
+        }
+      }
+    },
```

Comment:
Can you please try running `make clean build` and see if this section is updated to include the request and response schemas? (Same for openapi.yaml)